diff --git a/.tmux.conf b/.tmux.conf
index 5019e9b..f4cd6dd 100644
--- a/.tmux.conf
+++ b/.tmux.conf
@@ -1,5 +1,7 @@
 # use zsh as the default shell
-set-option -g default-command zsh
+# workaround for Mac OS X pasteboard, see <https://github.com/ChrisJohnsen/tmux-MacOSX-pasteboard>
+# don't pass -l, we don't want a login shell
+set-option -g default-command "reattach-to-user-namespace zsh"
 
 # change prefix key to C-t, I use C-b for backward-char (emacs)
 set-option -g prefix C-t
